weather.py

```diff
@@ -0,0 +1,14 @@
+class WeatherData:
```

Comment:
Docstrings missing in the whole file. Name is not self explanatory and it contains only one day's data.

Reply:
Name is not self explanatory and it contains only one day's data.